weave/trace_server/calls_query_builder/calls_query_builder.py

```diff
@@ -461,6 +461,8 @@ def set_hardcoded_filter(self, filter: HardCodedFilter) -> "CallsQuery":
         return self
 
     def add_order(self, field: str, direction: str) -> "CallsQuery":
+        if field in DISALLOWED_FILTERING_FIELDS:
+            raise ValueError(f"Field {field} is not allowed in ORDER BY")
```

Comment:
Is this error message propagated back to SDK users?

Reply:
we don't handle _any_ errors correctly on the streaming endpoint... my personal vendetta bug, but it's a big lift to fix.

Reply:
So here users will simply see some sort of 40x failure?

Reply:
hmm, yeah...